expression/expression.go

```diff
@@ -304,6 +304,7 @@ func ColumnInfos2ColumnsWithDBName(dbName, tblName model.CIStr, colInfos []*mode
 			continue
 		}
 		newCol := &Column{
+			FromID:   1,
```

Comment:
Why set it to `1` rather than other integers?

Reply:
It's a reasonable value but not a safe one. The mainly reason of this pr is to show @winoros 
 where the root cause is, due to we can not see each other tomorrow :)